.travis.yml

```diff
@@ -4,7 +4,8 @@ python:
     - "2.7"
 before_install:
     - "export DISPLAY=:99"
```

Comment:
You should be able to remove this `before_install` clause as well because the xvfb service will set this.